api/lib/source/polkadotV0-source.js

```diff
@@ -213,16 +213,88 @@ class polkadotAPI {
         validator.votingPower = 0
       }
     })
-
-    return allValidators.map((validator) =>
+    const validatorsWithoutProfiles = allValidators.map((validator) =>
       this.reducers.validatorReducer(this.network, validator)
     )
+    if (profile) {
+      return { validators: allValidators, validatorsWithoutProfiles }
```

Comment:
do not return different response formats

Reply:
why not?

Reply:
OK, actually it is very easy to change. Will take validators from store, simply

Reply:
OK, I know, it is better code :stuck_out_tongue: you're right